pkg/store/logentry/store.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package logentry
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/google/uuid"
+	"github.com/hyperledger/aries-framework-go/spi/storage"
+	"github.com/trustbloc/edge-core/pkg/log"
+	"github.com/trustbloc/vct/pkg/controller/command"
+
+	orberrors "github.com/trustbloc/orb/pkg/errors"
+)
+
+const (
+	nameSpace = "log-entry"
+
+	logTagName   = "Log"
+	indexTagName = "Index"
+)
+
+var logger = log.New("log-entry-store")
+
+// New creates db implementation of log entries.
+func New(provider storage.Provider) (*Store, error) {
+	store, err := provider.OpenStore(nameSpace)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open log entry store: %w", err)
+	}
+
+	return &Store{
```

Comment:
Do you need to configure the index tags?

Reply:
I do if I need to retrieve specific entry later (e.g. log 'A' + entry '1'). I am not 100% sure that I need this use case later on but it seems reasonable to be able to retrieve it. I can remove it later on if I don't need. I have to think about all the queries that we may need from log entries later on.